newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -149,7 +150,7 @@ export default class LocalPreviewLauncher extends React.Component<
         new gd.AbstractFileSystemJS(),
         localFileSystem
       );
-      const outputDir = path.join(fileSystem.getTempDir(), 'preview');
+      const outputDir = path.join(fileSystem.getTempDir(), `preview-${os.userInfo().username}`);
```

Comment:
I guess we don't need a user identifier here, because there is already one in `fileSystem.getTempDir()`?

Reply:
No I think we need an identifier because the issue is that on linux, when multiple users are connected to the machine at the same time, GDevelop will use the same directory for all users and this cause big issues like shared previews between users.

Reply:
Yes I understand that, but this identifier is already included in `fileSystem.getTempDir()`, so no need to have it here too, right?

Reply:
Uhm I mean this can be a second protection ? I know this is a little bit stupid but maybe this is not that bad to let preview-{username}

Reply:
Let's remove it, it's important to keep things as simple as they can be.

Reply:
@Oxey405 This is not solved yet :)